chia/cmds/coin_funcs.py

```diff
@@ -109,6 +109,7 @@ async def async_combine(args: Dict[str, Any], wallet_client: WalletRpcClient, fi
     excluded_amounts = args["excluded_amounts"]
     number_of_coins = args["number_of_coins"]
     max_dust_amount = Decimal(args["max_dust_amount"])
+    target_coin_amount = Decimal(args["target_coin_amount"])
```

Comment:
is `target_coin_amount` not optional? if it's `None`, passing it to `Decimal()` would fail, right?

Reply:
the default value is zero, that's what we normally use for cli.